Seasonal/Frostvale/HuntressMerge.cs

```diff
@@ -21,10 +25,14 @@ public class HuntressMerge
     private bool dontStopMissingIng = false;
 
     public void ScriptMain(IScriptInterface Bot)
-    {
+    {   
         Core.BankingBlackList.AddRange(new[] { "Sluagh Bell", "Punk Coal Elf Stabber", "Festive Punk Elf Stabber", "Wild Huntress' Sword " });
         Core.SetOptions();
 
+        if (!Core.isCompletedBefore(8448)){

```

Comment:
dont need the if just the MountOtziQuests line the void already has a check inside it 👍

Reply:
Removed the redundant conditional.